pkgs/development/python-modules/pydemumble/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  nanobind,
+  scikit-build-core,
+  cmake,
+  pytestCheckHook,
+  ninja,
+}:
+
+buildPythonPackage rec {
+  pname = "pydemumble";
+  version = "0.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "angr";
+    repo = "pydemumble";
+    tag = "v${version}";
+    hash = "sha256-JAUMTOYGHu64L0zLK2dzf0poHrGGiE29WoAR5kRsR+s=";
+    fetchSubmodules = true;
+  };
+
+  build-system = [
+    nanobind
```

Comment:
Ar you sure where are using this and not the submodule?

Reply:
good catch!

Reply:
But we want to use our nanobind and not the submodule, do we?

Reply:
i devendored it